src/main/java/domain/Car.java

```diff
@@ -0,0 +1,31 @@
+package domain;
+
+public class Car {
+
+    private final String name;
+    private int currentLocation;
+
+    public Car(String name) {
+        this.name = name;
+        this.currentLocation = 0;
+    }
+
+    public void move(boolean isMove) {
+        if (isMove) {
+            currentLocation += 1;
+        }
+    }
```

Comment:
```suggestion
    public void move(int number) {
        if (number >= MIN_LIMIT_NUMBER) {
            currentLocation += 1;
        }
    }
```
역할 상 CarRace의 isMovingForward를 Car가 가지고 있으면 어떨까하는데요
예를 들어, 차마다 다른 이동방식을 갖는다 라는 요구사항이 있을때도 조금 더 유연하게 변경이 가능할 거 같아요